src/sagemaker/cli/compatibility/v2/ast_transformer.py

```diff
@@ -18,12 +18,12 @@
 from sagemaker.cli.compatibility.v2 import modifiers
 
 FUNCTION_CALL_MODIFIERS = [
-    modifiers.predictors.PredictorConstructorRefactor(),
     modifiers.framework_version.FrameworkVersionEnforcer(),
     modifiers.tf_legacy_mode.TensorFlowLegacyModeConstructorUpgrader(),
     modifiers.tf_legacy_mode.TensorBoardParameterRemover(),
     modifiers.deprecated_params.TensorFlowScriptModeParameterRemover(),
     modifiers.tfs.TensorFlowServingConstructorRenamer(),
+    modifiers.predictors.PredictorConstructorRefactor(),
```

Comment:
this change is because the TFS constructor change for `Predictor` --> `TensorFlowPredictor` is covered, so we want to prevent false positives from `RealTimePredictor` --> `Predictor`

Reply:
+1 great call!